### PR TITLE
[CI] 【Hackathon 10th Spring No.20】功能模块 fastdeploy/engine/common_engine.py 单测补充

### PR DESCRIPTION
**Pull request by kesmeey**
## Motivation

功能模块 fastdeploy/engine/common_engine.py 单测补充
## Modifications
image

develop分支，覆盖率36%，miss行数765（75, 127-128, 169, 176-177, 200, 205, 215->219, 233-236, 247, 391, 393->426, 403->412, 440-534, 543-586, 592-593, 599, 606-652, 658-728, 735-801, 807-1080, 1098, 1101-1103, 1124-1125, 1133, 1143, 1148, 1151-1159, 1166-1181, 1184->1186, 1198-1206, 1209-1212, 1214->1217, 1221->1223, 1226->1231, 1228->1231, 1235, 1258-1281, 1296-1338, 1349-1357, 1369-1371, 1384-1390, 1398-1428, 1431-1434, 1437-1451, 1454-1465, 1471-1539, 1546-1670, 1685, 1688-1700, 1709-1743, 1760->1769, 1769->1783, 1783->1795, 1788->1795, 1853->1864, 1886->exit, 1932-1938, 1941, 1987->1989, 1989->1998, 1993, 1997, 1998->2001, 2053, 2101-2104, 2108->exit, 2117->2122, 2134, 2140->exit, 2155, 2185, 2193-2208, 2218->2220, 2224, 2231->2233, 2237, 2242-2243）

image

当前pr，覆盖率82%，miss行数186

完成覆盖行数 765-186=579

## Usage or Command

No need

## Accuracy Tests

No need
## Checklist

- [x] Add at least a tag in the PR title.
  - Tag list: [`[FDConfig]`,`[APIServer]`,`[Engine]`, `[Scheduler]`, `[PD Disaggregation]`, `[Executor]`, `[Graph Optimization]`, `[Speculative Decoding]`, `[RL]`, `[Models]`, `[Quantization]`, `[Loader]`, `[OP]`, `[KVCache]`, `[DataProcessor]`, `[BugFix]`, `[Docs]`, `[CI]`, `[Optimization]`, `[Feature]`, `[Benchmark]`, `[Others]`, `[XPU]`, `[HPU]`, `[GCU]`, `[DCU]`, `[Iluvatar]`, `[Metax]`]
  - You can add new tags based on the PR content, but the semantics must be clear.
- [x] Format your code, run `pre-commit` before commit.
- [x] Add unit tests. Please write the reason in this PR if no unit tests.
- [x] Provide accuracy results.
- [x] If the current PR is submitting to the `release` branch, make sure the PR has been submitted to the `develop` branch, then cherry-pick it to the `release` branch with the `[Cherry-Pick]` PR tag.

### CODE REVIEW

File: tests/engine/test_common_engine.py [hunk 1]
```diff
@@ -847,3 +932,109 @@ def __init__(self, *a, **k):
...
+        with self.assertRaises(EngineError) as ctx:
+            eng.insert_tasks([request])
+        self.assertIn("req1", str(ctx.exception))
+        if hasattr(eng, "_finalizer"):
+            try:
+                eng._finalizer.detach()
+            except Exception:
+                pass
+
+    def test_force_coverage_for_common_engine(self):
```
Comment (CSWYF3634076) [inline tests/engine/test_common_engine.py]: 这样不行，需要真正运行到才可以

**Review by CSWYF3634076 — APPROVED** [review]
LGTM

**Comment by CSWYF3634076** [comment]
0.6⭐️
@luotao1
